src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionService.ts

```diff
@@ -224,25 +219,41 @@ export class TerminalCompletionService extends Disposable implements ITerminalCo
 			}
 			const isDirectory = kind === TerminalCompletionItemKind.Folder;
 			const pathToResource = stat.resource.fsPath.replace(cwd.fsPath, '');
-			let label = pathToResource;
+
+			let label;
+			if (!prefix.startsWith('.')) {
```

Comment:
Many files and folders start with `.` so this will fall over there (`.git/`, `.bashrc`, etc.)